src/common.h

```diff
@@ -71,6 +71,17 @@ namespace sharp {
     std::string createNoiseType;
     double createNoiseMean;
     double createNoiseSigma;
+    std::string textValue;
+    std::string textFont;
+    std::string textFontfile;
+    int textWidth;
+    int textHeight;
+    VipsAlign textAlign;
+    bool textJustify;
+    int textDpi;
+    bool textRgba;
+    int textSpacing;
+    int autofitDpi;
```

Comment:
I know I originally suggested `autofitDpi` as a name so this is my fault, but perhaps we should rename this as `textAutofitDpi` throughout to keep in line with existing feature-related output properties such as `trimOffsetTop`.